### PR TITLE
Style/sudoku input

### PR DESCRIPTION
**Pull request by patrikglanemann**
Add styling to the input field.

### CODE REVIEW

File: src/NumberInput.css [hunk 1]
```diff
@@ -1,3 +1,30 @@
 .NumbInputField {
+  align-self: center;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+
   margin-top: 20px;
```
Comment (adrianammt) [inline src/NumberInput.css:6]: Wouldn't it be better to use rem here?

File: src/NumberInput.css [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+  padding: 0.5em 1.5em;
+
+  width: 95vw;
+  max-width: 360px;
+  height: 100px;
+
+  background: linear-gradient(300.63deg, #412027 45.56%, #562736 85.31%);
```
Comment (adrianammt) [inline src/NumberInput.css:13]: The gradient is a nice touch but it is very subtle. Styling is always subjective, but maybe make it a little bit more obvious?

File: src/NumberInput.css [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+  border-radius: 30px;
+}
+
+.NumbInputField__btn {
+  font-size: 2em;
+  color: white;
+  border: none;
+  border-radius: 10px;
+  background: none;
+  transition-duration: 0.2s;
```
Comment (adrianammt) [inline src/NumberInput.css:25]: I think you could add an effect here. Like ease in or out.

**Review by adrianammt — COMMENTED** [review]
I really like how it is developing!
I don't know if it is my screen, but when I click on a number and then on another one, a white line appears (see img).
Screenshot 2021-08-15 at 19 26 45

**Review by adrianammt — COMMENTED** [review]
I really like how it is developing!
I don't know if it is my scree, but when I click on a number and then on another one, a white line appears (I'll send the image via slack, my connection is still bad).

File: src/NumberInput.css (hunk 1 above)
Comment (mikemangas) [reply src/NumberInput.css:6]: i agree with adriana

File: src/NumberInput.css (hunk 2 above)
Comment (mikemangas) [reply src/NumberInput.css:13]: yes i agree with that too, adriana! very good point

**Comment by mikemangas** [comment]
> I really like how it is developing!
> I don't know if it is my screen, but when I click on a number and then on another one, a white line appears (I'll send the image via slack, my connection is still bad).

i dont have this bug @adrianammt

File: src/NumberInput.css (hunk 3 above)
Comment (mikemangas) [reply src/NumberInput.css:25]: i really like the hover effect.

File: src/NumberInput.css (hunk 1 above)
Comment (patrikglanemann) [reply src/NumberInput.css:6]: The gap is kind of provisional. But true, I still should have used rem.

File: src/NumberInput.css (hunk 2 above)
Comment (patrikglanemann) [reply src/NumberInput.css:13]: Oh no, I guess I have to talk to my designer again XD
Thanks you two for your input on that!

**Comment by patrikglanemann** [comment]
Uf, the white lines are inconvenient. But good to know that it is at least not on all screens/devices. I have to investigate.

**Pull request merged**